presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentListener.java

```diff
@@ -208,20 +211,27 @@ public void containerStopping(DockerContainer container, InspectContainerRespons
     static EnvironmentListener statsPrintingListener()
     {
         ScheduledExecutorService executorService = Executors.newScheduledThreadPool(2, daemonThreadsNamed("container-stats-%d"));
+        List<ScheduledFuture<?>> futures = new LinkedList<>();
```

Comment:
Linked?

Reply:
We are accessing values in a loop (no random gets). I can change it to ArrayList if you like.